ui/pages/permissions-connect/permissions-connect.component.js

```diff
@@ -318,7 +320,7 @@ export default class PermissionConnect extends Component {
 
     return (
       <div className="permissions-connect">
-        {this.renderTopBar()}
+        {hideTopBar ? null : this.renderTopBar()}
```

Comment:
```suggestion
        {!hideTopBar && this.renderTopBar()}
```

Reply:
I swear I did this originally and the linter complained. But it's not anymore 🤷‍♂️